script/Ciel/Day 15/15_01A_CIEL15A_1.txt

```diff
@@ -1151,31 +1151,31 @@ As we leave―――
 [sha:f136fb778eaa21a8b0449de160dd0542bbf784b6]{
 -- Page 70, Offset 41590.
 -- 後ろ髪を引かれて、もう一度だけ和風の建物を振り返る。
-I look back one last time at the Japanese-style building, as though reluctant to go.
+I take one last look at the building, as if reluctant leave it behind.
 }
 [sha:f16ec02bd81eb667c4f36c07de0d935ed8371358]{
 -- Page 70, Offset 41591.
 -- ……幻だろうか。
-I wonder if I'm just seeing things...
+Perhaps I'm just seeing things—――
 }
 [sha:964ee27665b14312524c4473c481b9eea46620ac]{
 -- Page 70, Offset 41592.
 -- 子供はじっと、何か言いたそうな目をして、走り去っていく俺の背中を見つめていた。
-The child is staring at me as I leave,
+But the child is staring at me as I leave,
  a look in his eyes that makes it seem like he had something he wanted to say.
 }
 [sha:6f0a065e2001a6781a31c27e8cc1ff79ef7e0461]{
 -- Page 71, Offset 41593.
 -- 旅客機が危なげなく滑走路に着陸するように、遠野志貴はもとの生活に戻っていた。
-Like an airplane landing safely on a runway, Tohno Shiki returns to his ordinary life.
+Like an airplane landing safely on the runway, Tohno Shiki returns to his ordinary life.
 }
 [sha:b7ce5e03eb6dfcc91cbafaccc94d85c54e2f8ccf]{
 -- Page 71, Offset 41594.
 -- 屋敷に帰って、秋葉に小言を言われて、朝になったら翡翠に起こされて、琥珀さんの朝食をいただいて屋敷を出る。
 I go back to the mansion,
- get scolded by Akiha,
+ get nagged at by Akiha,
```

Comment:
I suppose when it's just "nagging"; yes, but I think getting "nagged at" is a bit different. I don't remember if I ever mentioned this, but Akiha's 小言 is something that comes up a lot with her. It's her chiding/pestering Shiki over minutiae, rather than him getting seriously chewed out. I think 'nag' works best (because it has the same nuance of 'being annoying about small things' as 小言 does), but maybe there's something else.

Reply:
I think my main problem is that I don't think "nagged at" in this use is a correct construction - maybe 
> get an earful of Akiha's nagging

or something?

Reply:
I don't see why 'nagged at' wouldn't be valid?

It's well attested, as far as I can tell.

Reply:
Nagged at is definitely valid, but in this specific context it feels a little off. I'll run it by some other natives to see if I'm just crazy.

Reply:
Yeah consensus is "that's not entirely wrong but also not right"

Reply:
Other input - nag has mild misogynistic connotations, scold more parental. Pester honestly might be good, implies some of the small details, though has less of the chide/kvetch included. Get lectured by Akiha?

Reply:
Wouldn't chide be best at that point?
The misogynistic angle isn't really something I considered, because honestly it feels fairly similar in JP in terms of its usage (小言 is usually applied to mothers trying to get their lazy son to do their homework, etc) but they don't really think of misogyny in the same way.

Reply:
Chide works for me